### ISSUE TITLE
when adding dropdownOptions showFlags and showDialCodeInSelection, auto search does not work

### ISSUE THREAD

**Issue by michal-gordon** [iamstevendao/vue-tel-input#362]
When I enable showFlag and showDialCodeInSelection, the country list does not auto scroll according to letters pressed on keyboard. 
When i remove the dropdownOptions object, it works as expected. but i need the showDialCodeInSelection to be true. 

```
<vue-tel-input
    v-model="phoneNumber"
    :class="{ '--invalid': isInvalid }"
    :enabled-country-code="true"
    @validate="validatePhone"
    @country-changed="countryChanged"
    :dropdownOptions="{
      showFlags: true,
      showDialCodeInSelection: true,
    }"
  />
```

**Comment by FeistyMango** [comment]
Confirmed this is the same issue I ran into as well.

**Comment by Rahul-Nlab** [comment]
Yes, even a single config option such as placeholder would break the auto search feature for country, and I don't see any solution to this issue. I'm trying to implement the search box with auto-focus for searching countries now as a fall back mechanism.
